Calibration/HcalAlCaRecoProducers/python/alcaIsoTracksFilter_cff.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+
+from Calibration.HcalAlCaRecoProducers.alcaIsoTracksFilter_cfi import *
+
+from Configuration.Eras.Modifier_run2_ECAL_2017_cff import run2_ECAL_2017
```

Comment:
It could be Hcal2017 as well. Since these are thresholds for EE, I made it for Ecal. But there is no Ecal2018. If you want I can make both of them Hcal2017 and Hcal2018. The current setup works for 2016, 2017, 2018 and that is the purpose of this PR. We want to port it to 10_4 as well once it is merged to master.

Reply:
Please let me know if the current form is acceptable or do I change Ecal2017 to Hcal2017? Waiting for your comment.

Reply:
@tocheng Please let me know your views - I replied to your question

Reply:
@bsunanda I'd like to use HCAL_2017 cos the parameters are filters from ECAL not parameters applied to ECAL reconstruction.

Reply:
@tocheng OK - I am resubmitting